test/functional/module_stream/list_test.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/module_stream'
+
+module HammerCLIKatello
+  describe ModuleStreamCommand::ListCommand do
+    it 'allows minimal options' do
+      api_expects(:module_streams, :index)
+
+      run_cmd(%w(module-stream list))
+    end
+
+    it 'can be provided by repository ID' do
+      run_cmd(%w(module-stream list --repository-id 1))
+    end
+
+    it 'product params needed with repository name' do
+      cmd = run_cmd(%w(module-stream list --repository Repo))
```

Comment:
missing expectation

Reply:
What is the expectation for the API call here? I'm not seeing an API call being made

```
[vagrant@hammer-devel2 ~]$ hammer -d  module-stream list --repository zoo_with_modules
[ INFO 2018-09-04T14:56:57 Init] Initialization of Hammer CLI (0.14.pre.develop) has started...
[DEBUG 2018-09-04T14:56:57 Init] Running at ruby 2.4.3-p205
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli_config.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/csv.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/foreman.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/foreman_admin.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/foreman_admin_logging_core.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/foreman_admin_logging_katello.yml has been loaded
[ INFO 2018-09-04T14:56:57 Init] Configuration from the file /home/vagrant/.hammer/cli.modules.d/katello.yml has been loaded
[ INFO 2018-09-04T14:56:57 SSLoptions] Matching CA cert was found in local CA store /home/vagrant/.hammer/certs/coffee.jomitsch.example.com_443.pem
[DEBUG 2018-09-04T14:56:57 SSLoptions] SSL options: {
     :verify_ssl => true,
    :ssl_ca_file => "/home/vagrant/.hammer/certs/coffee.jomitsch.example.com_443.pem"
}
[DEBUG 2018-09-04T14:56:57 API] Global headers: {
        :content_type => "application/json",
              :accept => "application/json;version=2",
    "Accept-Language" => "en"
}
[DEBUG 2018-09-04T14:56:57 API] Follow redirects: never
[DEBUG 2018-09-04T14:56:57 Connection] Registered: foreman
[ INFO 2018-09-04T14:56:57 Modules] Extension module hammer_cli_csv (2.3.1) loaded.
[ INFO 2018-09-04T14:56:57 Modules] Extension module hammer_cli_foreman (0.14.pre.develop) loaded.
[ INFO 2018-09-04T14:56:57 Modules] Extension module hammer_cli_foreman_admin (0.0.8) loaded.
[ INFO 2018-09-04T14:56:57 HammerCLI::MainCommand] subcommand organization (HammerCLIForeman::Organization) was removed.
[ INFO 2018-09-04T14:56:57 HammerCLI::MainCommand] subcommand organization (HammerCLIKatello::Organization) was created.
[ INFO 2018-09-04T14:56:57 Modules] Extension module hammer_cli_katello (0.14.0) loaded.
[DEBUG 2018-09-04T14:56:57 Init] Using locale 'en'
[DEBUG 2018-09-04T14:56:57 Init] 'mo' files for locale domain 'hammer-cli' loaded from '/home/vagrant/hammer-cli/locale'
[DEBUG 2018-09-04T14:56:57 Init] 'mo' files for locale domain 'hammer-cli-foreman' loaded from '/home/vagrant/hammer-cli-foreman/locale'
[DEBUG 2018-09-04T14:56:57 Init] 'mo' files for locale domain 'hammer-cli-csv' loaded from '/home/vagrant/hammer-cli-csv/locale'
[DEBUG 2018-09-04T14:56:57 Init] 'mo' files for locale domain 'hammer-cli-katello' loaded from '/home/vagrant/hammer-cli-katello/locale'
[ INFO 2018-09-04T14:56:57 HammerCLI::MainCommand] Called with options: {"option_debug"=>true}
[ INFO 2018-09-04T14:56:57 HammerCLIKatello::ModuleStreamCommand] Called with options: {}
[DEBUG 2018-09-04T14:56:57 Exception] Using exception handler HammerCLIKatello::ExceptionHandler#handle_usage_exception
[ERROR 2018-09-04T14:56:57 Exception] Error: At least one of options --product, --product-id is required.

See: 'hammer module-stream list --help'.
Error: At least one of options --product, --product-id is required.

See: 'hammer module-stream list --help'.
[ERROR 2018-09-04T14:56:57 Exception] 

```

Reply:
```rb
api_expects_no_call
```